Core/PrimaryGenerator.cpp

```diff
@@ -61,63 +61,18 @@ G4PrimaryVertex* PrimaryGenerator::GenerateThreeGammaVertex(
   Double_t weight;
   Double_t weight_max = event.GetWtMax() * pow(10, -1);
   Double_t rwt;
-  Double_t M_max = 7.65928 * pow(10, -6);
-  do 
+  Double_t M_max;
+  if(channel==MaterialExtension::DecayChannel::Ortho3G or channel == MaterialExtension::DecayChannel::Direct)
```

Comment:
Please change "or" to "||"

Reply:
I agree, || is used more (for example MaterialExtension.cpp or HistoManager.cpp). It would be better to preserve the same convention. In addition, could you fix spaces between the first "==" so it will look like  "channel == MaterialEx..."